### ISSUE TITLE
Undefined variable: loop

### ISSUE THREAD

**Issue by reidsneo** [esyede/single-blade#1]
The library is great! instead of loading illuminate dependency just load a single library can handle almost everything, but still it unable to process $loop function, loop function is belong to essentials part when develop a feature such breadcrumb and styling part

```blade
@foreach ($users as $user)
    @if ($loop->first)
        This is the first iteration.
    @endif

    @if ($loop->last)
        This is the last iteration.
    @endif

    <p>This is user {{ $user->id }}</p>
@endforeach
```

`Undefined variable: loop`

Reference : https://laravel.com/docs/5.3/blade#the-loop-variable

**Comment by esyede** [comment]
@reidsneo  Yep. there is no `$loop` functionality atm. It's a nice to have feature indeed, but I've been busy lately so I can't promise you on this.
Can you kindly send a PR?

**Comment by esyede** [comment]
Closed in 50ff46b436e6ca74ee7726b344c628932453edb2
